### ISSUE TITLE
Editor: Formatting lost after going back to "HTML" mode from "Visual" mode

### ISSUE THREAD

**Issue by hemantg05** [infor-design/enterprise#2424]
**Bug Description**
Editor formatting is lost after going back to "HTML" mode from "Visual" mode.

**Steps to reproduce the behavior**
Refer to the following GIF to reproduce the behaviour.
![Formatting_Lost](https://user-images.githubusercontent.com/1594815/59929476-2f5e3900-9441-11e9-9a65-0074a6b0c4c6.gif)

**Expected behavior**
Formatting should remain for user to be able to conveniently edit the markup in HTML mode.

**Version**
- ids-enterprise: [all versions]

**Screenshots**
Please see GIF above.

**Platform**
Tested and found on Chrome. It should be reproduced almost on every browser/device.

**Comment by tmcconechy** [comment]
Would you be able to paste in that markup used for testing?

**Comment by hemantg05** [comment]
Sure, here is the markup.
```
<tbody>
    <tr>
        <td class="label" style="padding: 15px 0 15px 0;padding-left: 15px;">{{orderRoundingOffText}}</td>
        <td class="value" style="padding: 15px 0 15px 0;text-align: right;padding-right: 15px;">{{roundingOff}}</td>
    </tr>
    <tr>
        <td class="total-label" style="padding: 15px 0 15px 0;padding-left: 15px;padding-right: 15px;font-size: 18px;"> <strong>{{orderTotalText}} ({{currencyCode}})</strong> </td>
        <td class="total-value" style="padding: 15px 0 15px 0;text-align: right;padding-right: 15px;padding-left: 15px;font-size: 18px;"> <strong>{{total}}</strong> </td>
    </tr>
</tbody>
```

**Comment by brianjuan** [comment]
I'm sorry it seems that I did not submit my comment yesterday on why I failed this ticket.
http://master-enterprise.demo.design.infor.com/components/editor/test-source-formatter.html
In **Firefox**, the HTML view gets broken after you do "Clear Formatting".
Steps:
1. Open the page in Firefox, then go to Visual view
2. Highlight all then click "Clear Formatting"
3. Go back to HTML view, see issue: it's broken now.
![image](https://user-images.githubusercontent.com/39508864/64015052-3f3b7d00-cb56-11e9-952e-e1e8b79b1dce.png)
